### ISSUE TITLE
GraphQL Microservice question

### ISSUE THREAD

**Issue by shivshankardayal** [tiangolo/fastapi#3196]
I want to implement GraphQL microservice based on FastAPI. The problem I am facing is how does microservices update the  service registry after startup event has happened. This is going to be a server side service discovery architecture.

**Comment by Hultner** [comment]
Not sure if this is related to FastAPI, seems more like an internal architectural pattern.

But I'd suppose generally you would send whatever registration request you use to the service registry in that startup code?

**Comment by tiangolo** [comment]
Agreed this is probably not really a FastAPI question... although I'm not sure I fully understand the question.

Thanks for the help @Hultner! :coffee: 

For GraphQL, you might want to check Strawberry: https://strawberry.rocks/

If you still have questions, please let me know what are you intending to solve, and please add a self-contained, [minimal, reproducible, example](https://stackoverflow.com/help/minimal-reproducible-example) that I can copy-paste to replicate it.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
